### ISSUE TITLE
The problem of receiving the notification of connecting and disconnecting the device in Ntfy

### ISSUE THREAD

**Issue by bravery-only** [Ylianst/MeshCentral#4723]
I enabled Nfty, but I don't receive notifications when the device is connected and disconnected. Is this option not Active yet?

![Ntfy](https://user-images.githubusercontent.com/42680639/200178240-8597934d-08bf-4ede-828d-76044db2eae3.jpg)

**Comment by si458** [comment]
**Plz can u start submitting these issues as questions, this is not a bug!**

What version do u have installed?

**Comment by bravery-only** [comment]
Version 10.97
I am testing the device through the box below the message and there is no problem. I just disconnect and connect the device and do not receive the notification
![Messaging](https://user-images.githubusercontent.com/42680639/200179722-6ef6a6a1-fdba-4eb0-b9bc-bfc8f3b51f96.jpg)

**Comment by si458** [comment]
The device connection and siconnect messaging notifications are for when the devices connect to your server and disconnect from you server

The send message picture you have shown is to send pop up messages to the device itself. Nothing to do with messaging

**Comment by bravery-only** [comment]
I meant this with this Box NTF tested. Will not be notified to disconnect and connect.

**Comment by bravery-only** [comment]
I changed the notification timer to 1 second. There is Internet access on both servers. Only the server in local mode that I installed for testing did not receive a notification

**Comment by barart** [comment]
I have a similar problem... the notification (connect/discconect) does not work, but i can send messages

**Comment by mschubeck-redux** [comment]
> I changed the notification timer to 1 second.

@bravery-only Can you share how you changed the notification timer?

**Comment by gasparakos** [comment]
I have the same experience, no notification on device connect, disconnect in Ntfy. On user page send message is working. Our server in hybrid mode:
```
   "WANonly": false,                                                                         
   "LANonly": false,    
```

**Comment by brahunafisk** [comment]
I have the same issue.  I can message my device no problem but will not receive connect/disconnect.  E-mail works every 5 minutes as it should.

**Comment by barart** [comment]
Any update on this?

**Comment by brahunafisk** [comment]
> Any update on this?

It's working after the upgrade...did you build yourself a new server?  I did and I just had to boot it.  My original problem was with the config.  Ntfy is working great since.

**Comment by barart** [comment]
> It's working after the upgrade...did you build yourself a new server? I did and I just had to boot it. My original problem was with the config. Ntfy is working great since.

Yes, i have been upgrading it, at the moment im on latest version (1.1.4) and it still not working, i receive the two factor login code on ntfy and the test message without any problems, but the connected/disconnected notifications still not working

**Comment by brahunafisk** [comment]
I’d recommended looking at the config file and look for spaces around the end.  Having a line space was my issue when I had your exact problem.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by barart** [comment]
> I’d recommended looking at the config file and look for spaces around the end. Having a line space was my issue when I had your exact problem. 

I already check and recheck the config file and it dont have spaces on lines or extra blank lines between them my messaging config is exactly like this:

```
"messaging": {
    "ntfy": {
      "host": "myntfyserver.domain.com",
      "userurl": "https://www.domain.com"
    }
  },
```

**Comment by brahunafisk** [comment]
Here’s Mine:


   },
   "messaging": {
    "ntfy": true,
    "__ntfy": {
    "host": "ntfy.sh",
    "userurl": "https://myserver.com/userhelp"
    }

**Comment by si458** [comment]
I don't get these notifications with pushover

So I think the is a bug in general with the notifications

**Comment by brahunafisk** [comment]
Mine work great.  Got some today when a server went offline.

**Comment by barart** [comment]
Still not working on the latest version, this issue must be opened again

**Comment by si458** [comment]
We haven't released an update yet, so it is not included in 1.1.11 but 1.1.12 when we release it
